pagespeed/automatic/system_test_helpers.sh

```diff
@@ -825,3 +825,51 @@ function kill_listener_port {
   PORT="$2"
   kill -9 $(lsof -t -i TCP:${PORT} -s TCP:LISTEN -a -c "/^${CMDLINE}$/") || true
 }
+
+# Performs timed reads on the output from a command passed via $1. The stream
```

Comment:
re: Any reason to have this here and not just follow_flushes.sh?
companion PR for nps: https://github.com/pagespeed/ngx_pagespeed/pull/1217
(updated the commit description to mention that too)